src/app/zap-templates/zcl/data-model/chip/time-synchronization-cluster.xml

```diff
@@ -28,8 +28,33 @@ limitations under the License.
     <description>Accurate time is required for a number of reasons, including scheduling, display and validating
 security materials.</description>
     <!-- Base data types -->
+    <attribute side="server" code="0x0000" define="UTC_TIME" type="EPOCH_US" writable="false" optional="false">UTCTime</attribute>
```

Comment:
Needs to be marked nullable.  So do a bunch of other attributes here.

Reply:
Added isNullable in various places.